components/pkg-export-docker/src/docker.rs

```diff
@@ -258,6 +268,11 @@ impl<'a> DockerImage {
             .arg(credentials.username)
             .arg("--password")
             .arg(credentials.password);
+
+        if let &Some(arg) = registry_url {
```

Comment:
Making the above changes means you should be able to remove the `&` here and just have `Some`